articles/aks/dapr-settings.md

```diff
@@ -241,7 +241,10 @@ az k8s-extension update --cluster-type managedClusters \
 
 ## Meet network requirements
 
-The Dapr extension for AKS and Arc for Kubernetes requires outbound URLs on `https://:443` to function. In addition to the `https://mcr.microsoft.com/daprio` URL for pulling Dapr artifacts, verify you've included the [outbound URLs required for AKS or Arc for Kubernetes](../azure-arc/kubernetes/network-requirements.md). 
+The Dapr extension for AKS and Arc for Kubernetes requires the following outbound URLs on `https://:443` to function:
+1. `https://mcr.microsoft.com/daprio` URL for pulling Dapr artifacts.
+2. `https://linuxgeneva-microsoft.azurecr.io/` URL for pulling some Dapr dependencies.
```

Comment:
```suggestion
1. `https://linuxgeneva-microsoft.azurecr.io/` URL for pulling any Dapr dependencies.
```

Reply:
hmm, I think `some` is better here, as `any` would reflect that all dependencies are resolved linuxgeneva registry, which is not the case.

Reply:
ahh good callout, thank you!